Wrapper shell scripts for starting and stopping services

getplank opened the pull request:
Start and stop scripts (for *nix environments) to automate the instructions provided at https://github.com/bitpay/bitcore-wallet-service/blob/master/installation.md

If this is accepted, I can update the documentation as well.

matiu commented:
Yes! thanks for the contribution!

Could you update the docs and also add 'start.sh' to package.json ->  npm start?

getplank commented:
Done. Also added a "stop" target to package.json

All doc changes in installation.md. Cleaned up some inaccurate npm references there as well. The installation document is completely accurate and tested.

isocolsky commented on installation.md:
```diff
@@ -1,15 +1,42 @@
+BWS is composed of 5 separate node services -
+Locker - locker/locker.js
+Message Broker - messagebroker/messagebroker.js
+Blockchain Monitor - bcmonitor/bcmonitor.js (This service talks to the Insight service hosted by Bitpay)
```
Actually, `bcmonitor` will use whatever instance is specified in `config.js` under `blockchainExplorerOpts`

getplank replied:
corrected

isocolsky commented:
LGTM
This is very useful, thank you!
